private.h

```diff
@@ -97,6 +99,10 @@ struct _DroidMediaBufferQueue : public android::RefBase {
 
   DroidMediaBufferSlot m_slots[android::BufferQueue::NUM_BUFFER_SLOTS];
 
+  // Storage for buffers that have their resources cleaned up, but the client still holds a raw reference.
+  // The first part is used to identify the raw pointer from client, the second to actually clean the memory.
+  std::unordered_map<DroidMediaBuffer*, android::sp<DroidMediaBuffer>> m_unslotted;
```

Comment:
This causes an error on older Android bases because unordered_map is not defined, probably a missing header?

Reply:
Added `#include <unordered_map>` above, let me know if this fixes the error.